staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/mutating_work_estimator.go

```diff
@@ -24,27 +24,36 @@ import (
 	apirequest "k8s.io/apiserver/pkg/endpoints/request"
 )
 
+const (
+	watchesPerSeat          = 10.0
+	eventAdditionalDuration = 5 * time.Millisecond
```

Comment:
I like "notification" better than "event", since we also have a different thing called "event" and "notification" is a pretty standard and more specific term for what we are talking about here.

Reply:
The terminology that is used everywhere is "watch event", not "watch notification".
So we should be consistent with the terminology that is used across the project (even if we believe that notification would be a better one).

Reply:
Thanks, I had forgotten about "watch event".